src/lib/pages/start-workflow.svelte

```diff
@@ -64,6 +64,29 @@
 
   let searchAttributes: SearchAttributeInput[] = [];
 
+  $: errorWorkflowDetails = extractWorkflowFromError(error);
+
+  function extractWorkflowFromError(errorMessage: string): {
+    workflowId?: string;
+    runId?: string;
+  } {
+    if (!errorMessage) return {};
+
+    const workflowIdMatch = errorMessage.match(
+      /workflow[_\s]*id[:\s=]+['"]*([^'"\s,;).]+)['"]*[.\s]*/i,
```

Comment:
This regex isn't going to work because workflowIds/runIds can have periods in them.

So something like workflowId=temporal.canary.cron-workflow.sanity-2025-08-28T09 would go to just temporal/runId

Might need to try to split the errorMessage at WorkflowId: and RunId:

Reply:
Okay, I added in the string functions instead of the regex. I think it's better now, but let me know what you think and if there are any other edge cases I should be keeping an eye out for. Thanks boss!